controller.rst

```diff
@@ -148,6 +148,11 @@ that's available to you with or without the use of the base
 action is to look in the
 :class:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller` class.
 
+.. tip::
+    If you know what you're doing, you can alternatively extend
```

Comment:
~~If you know what you're doing, ~~ : this just creates fear to me, whereas the diff is explained very simply in the next sentence, no?